requirements.txt

```diff
@@ -18,3 +17,7 @@ gitpython~=3.1.11
 docker~=4.1.0
 yq~=2.11.1
 jq~=1.1.1
+s3pypi~=0.11.0
```

Comment:
Is this return of the s3pypi dependency intentional, we once skipped it in https://github.com/nordic-institute/X-Road-Security-Server-toolkit/pull/19

Still the resolver cannot resolve this dependencies with known good compatibility versions:
 ```
  ERROR: s3pypi 0.11.0 has requirement wheel<0.34.0,>=0.33.6, but you'll have wheel 0.34.2 which is incompatible.
  ```
  
As we still have no code that does anything with s3 it seems we should not include that dependency.